docs/direct-access-urls.md

```diff
@@ -142,16 +154,16 @@ The following endpoints can be used to obtain direct access URLs:
 * `/nodes/{nodeId}/versions/{versionId}/request-direct-access-url`
```

Comment:
Is this complete ? eg.

`POST /nodes/{nodeId}/request-direct-access-url`
`POST /nodes/{nodeId}/renditions/{renditionId}/request-direct-access-url`

Maybe also prefix others with POST ?

Also @SaraAspery may also separately update again with new (if implemented):

`POST /nodes/{nodeId}/versions/{versionId}/renditions/{renditionId}/request-direct-access-url`

Reply:
I recon that only versions-renditions was missing which I added. 
About adding POST prefix to all - I guess that just in the next line after `Paths` there is a paragraph explaining that all of them are POST methods:
`**Method:** **`POST`**`

Reply:
Thanks for that :)